instaclustr/resource_vpc_peering.go

```diff
@@ -187,3 +199,25 @@ func resourceVpcPeeringStateImport(d *schema.ResourceData, meta interface{}) ([]
 	d.Set("vpc_peering_id", idParts[1])
 	return []*schema.ResourceData{d}, nil
 }
+
+func createVpcPeeringRequest(d *schema.ResourceData) (CreateVPCPeeringRequest, error) {
+	result := CreateVPCPeeringRequest{}
+	if _, isSet := d.GetOk("peer_subnet"); isSet {
```

Comment:
would it be possible to just convert the single subnet into an array of subnets after validation and just handle it as a list of "subnets"? - not sure if this is good to do, but it does keep validation the same for both which sounds good to me?